assets/sass/_top-button.scss

```diff
@@ -0,0 +1,26 @@
+.top-button {
+    border-radius: 3px;
+    color: #fff!important;
+    line-height: 20px;
+    transition: all .5s ease-in-out;
+    background: linear-gradient(180deg,#90a4ae,#78909c);
+    width: 40px;
+    height: 25px;
+    position: fixed;
+    right: 27%;
+    bottom: 40px;
+    opacity: 30%;
+    z-index: 1000001;
+    cursor: pointer;
+    text-align: center;
+}
+
+.top-button::after {
+    content: "🔝";
```

Comment:
I am wondering if a simple "chevron" pointing up would be better?

Reply:
Yes, I wanted to use a chevron and tried, but couldn't find one. I also tried using a font-awesome one, but it wouldn't render. It seems it is not available in our current font awesome set.

Reply:
We do use Font Awesome Free for icons on this site, but there are some arrows in here that might suffice: https://fontawesome.com/icons?d=gallery&q=arrow%20up